main/src/main/scala/sbt/coursierint/LMCoursier.scala

```diff
@@ -19,11 +20,15 @@ import lmcoursier._
 import lmcoursier.credentials.Credentials
 import Keys._
 import sbt.librarymanagement._
+import sbt.librarymanagement.ivy.{ Credentials => IvyCredentials }
 import sbt.util.Logger
 import sbt.io.syntax._
 import xsbti.AppConfiguration
 
 object LMCoursier {
+  private[sbt] val credentialRegistry: mutable.Map[(String, String), IvyCredentials] =
```

Comment:
Could this be `private` or `private[this]`. It doesn't seem like it's referenced outside this file. Also, for mutable maps, I tend to default to `ConcurrentHashMap` since it seems to offer a better performance/thread safety combination than scala.mutable.Map. My guess is performance doesn't matter here, but there might be some concurrency.

Reply:
Both very good points.

Reply:
9755234